libs/RobotKit/include/RobotController.h

```diff
@@ -191,6 +193,18 @@ class RobotController : public interface::RobotController
 
 	void startDisconnectionBehavior() final { stopActuators(); }
 
+	void startAutonomousActivity() final
```

Comment:
de même que pour en dessous, le nom n'est pas très clair. en soi on ne commence pas une activité ici, on se met dans le mode autonomous activities

```suggestion
	void startAutonomousActivitiesMode() final
```